projects/uart_test/src/main.c

```diff
@@ -0,0 +1,66 @@
+// x86 UART initiate receiver and transmitter
```

Comment:
Why not make this test under `libraries/ms-common/test/`?

First you can use `TEST_ASSERT_*` to validate rather than having to run this. Also it would run the test on Travis CI?